### DISCUSSION TITLE
Are there any tricks to force fixed width when printing digits?

### DISCUSSION THREAD

**Discussion by kiddingme1** [Bodmer/TFT_eSPI#2228] [Q&A - Fonts]
For faster changing data it is easier to read on the display if the numbers do not shift horizontally around due to the varying width of each digit. Orbitron font for reference. I tried messing with width, xAdvance, and xOffset within the Orbitron font header file but it didn't seem to have any effect. 

Would love to have fixed font width on digits 0-9.

**Comment by JimDrewGH** [comment]
You can make a fixed width font yourself.  I think there are a few already for TFT_eSPI as well.  I have converted some 8 bit style monospaced TrueType fonts to the smooth font format that TFT_eSPI uses.

I found this info on how to use the smooth fonts, and it works perfectly:

https://wiki.seeedstudio.com/Wio-Terminal-LCD-Anti-aliased-Fonts/

**Comment by kiddingme1** [comment]
Oops, I figured out why the changes to xAdvanced were not having an effect...I was editing the wrong font header file. My bad. I should be able to achieve the desired effect now. Thanks.
